frontend/src/modules/entitydetails/components/Metadata.js

```diff
@@ -66,7 +66,14 @@ export default class Metadata extends React.Component<Props> {
         // $FLOW_IGNORE
         if (e.target && e.target.classList.contains('placeable')) {
             // $FLOW_IGNORE
-            if (e.target.childNodes.length) {
+            if (e.target.dataset['match']) {
+                this.props.addTextToEditorTranslation(
+                    // $FLOW_IGNORE
+                    e.target.dataset['match']
+                );
+            }
+            // $FLOW_IGNORE
+            else if (e.target.childNodes.length) {
```

Comment:
I wonder if it'd be cleaner to require `data-match` attribute for all placeable types and drop this `else`.

Reply:
That's kinda useless on most of the parsers. I'd rather make the API as clean as possible, and make the code underneath a bit more clunky in such a case. If you feel strongly about it though, I'm open to changing it.

Reply:
It also adds duplicated to the DOM, not sure how much that costs but I'm thinking we might as well stay as slim as we can (says the guy who uses React... ).